src/config/test-config.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
   clusterEnv: 'test',
   awsRegion: 'eu-west-1',
   awsAccountIdPromise: getAwsAccountId,
-  workerInstanceConfigUrl: 'https://raw.githubusercontent.com/biomage-ltd/iac/master/charts/worker-instance.yaml',
+  workerInstanceConfigUrl: 'https://raw.githubusercontent.com/biomage-ltd/iac/master/releases/production/worker.yaml',
+  pipelineInstanceConfigUrl: 'https://raw.githubusercontent.com/biomage-ltd/iac/master/releases/production/pipeline.yaml',
```

Comment:
why do we need these for the test config?

Reply:
These are for defining things like the most up-to-date image for the given deployment, etc. The API uses that data to inform, e.g., which image to launch the pipeline server with. You can see this as part of a `ConfigMap` in the files.

These are only created for staging and production when the specific projects are deployed. We can use the production one for testing purposes so we always have some data in the files -- the actual data itself does not matter, because we don't actually call to any services, they just have to be defined.